server/data/data-library/src/test/java/org/apache/james/rrt/lib/AbstractRecipientRewriteTableTest.java

```diff
@@ -334,10 +334,46 @@ public void removeGroupMappingShouldDelete() throws Exception {
         virtualUserTable.removeMapping(source, Mapping.group(ADDRESS));
         virtualUserTable.removeMapping(source, Mapping.group(address2));
 
-        assertThat(virtualUserTable.getMappings(USER, domain))
+        assertThat(virtualUserTable.getUserDomainMappings(source))
             .isEqualTo(MappingsImpl.empty());
     }
 
+    @Test
+    public void addAliasMappingShouldStore() throws Exception {
+        Domain domain = Domain.LOCALHOST;
+        String address2 = "test@james";
+        MappingSource source = MappingSource.fromUser(USER, domain);
+
+        virtualUserTable.addMapping(source, Mapping.alias(ADDRESS));
+        virtualUserTable.addMapping(source, Mapping.alias(address2));
+
+        assertThat(virtualUserTable.getUserDomainMappings(source)).hasSize(2);
```

Comment:
We generally tend to write more precise asserts, for instance, **containsOnly** that details the content of the returned collection.

We could do that by having mappings defined in variables.

Again, I propose modify other tests to slightly enhance that test class.